config/locales/api/help/en.html

```diff
@@ -114,7 +114,7 @@ <h3>Class chapters</h3>
             <h3>Tele-consultation training</h3>
             <p>If your hospital is using Simple for tele-consultation, this training is for you.</p>
             <div class="video-small">
-                <a href="https://youtu.be/LpLeFFKhXd4">Tele-consultation with Simple<span>English</span></a>
+                <a href="https://youtu.be/aI1bGpS08oY">Tele-consultation with Simple<span>English</span></a>
```

Comment:
Should this be `ExternalLinksHelper::TELEMEDICINE_TRAINING_ENGLISH_YOUTUBE_LINK`?